src/animals/Eagle.ts

```diff
@@ -0,0 +1,41 @@
+ import { AnimalCandidate } from "../hierarchy/Animal"
+import Birds from "../hierarchy/Birds"
+import Reptile from "../hierarchy/Reptile"
+import { ICanFly } from "../interfaces/ICanFly"
+import { ICanTalk } from "../interfaces/ICanTalk"
+const EAGLE_SPACE_NEEDED_AS_ADULT = 100
+const EAGLE_SPACE_NEEDED_AS_CHILD = 50
+const NUMBER_OF_BABY_EAGLES_BORN = 4
+const AGE_TO_BE_ADULT = 1;
```

Comment:
Since all the birds have the same age to be adult maybe we add the constant in the birds class and also maybe name it something like BIRDS_AGE_TO_BE_ADULT so we can easily know its for birds and not be confused with the other age to be adult which is set to 2 in the Animal class

Reply:
I thought about it, but initially it was the same with mammals, then we changed the implementation, what if another type of bird is added where the age is 4, for instance.